bigquery_etl/cli/monitoring.py

```diff
@@ -144,6 +156,164 @@ def deploy(
     )
 
 
+def _sql_rules_from_file(custom_rules_file, project, dataset, table) -> list:
+    """Extract the SQL rules from the custom rules file."""
+    jinja_params = {
+        **{
+            "dataset_id": dataset,
+            "table_name": table,
+            "project_id": project,
+        },
+    }
+    if "format" not in jinja_params:
+        jinja_params["format"] = False
+
+    rendered_result = render_template(
+        custom_rules_file.name,
+        template_folder=str(custom_rules_file.parent),
+        templates_dir="",
+        **jinja_params,
+    )
+
+    statements = []
+    for statement in sqlglot.parse(rendered_result, read="bigquery"):
+        if statement is None:
+            continue
+
+        for select_statement in statement.find_all(sqlglot.exp.Select):
+            statements.append(select_statement)
+
+    return statements
+
+
+@monitoring.command(
+    help="""
+    Deploy custom SQL rules.
```

Comment:
I wonder if maybe we should rename the command to something like `custom_monitor` to indicate a lot more clearly the resource deployed?

Reply:
ignore this please, I just realised the actual command would be:
`./bqetl monitoring deploy_custom_rules`